src/query/ast/src/ast/query.rs

```diff
@@ -119,9 +119,31 @@ pub enum SelectTarget<'a> {
         alias: Option<Identifier<'a>>,
     },
 
-    // Qualified name, e.g. `SELECT t.a, t.* FROM t`.
+    // Qualified name, e.g. `SELECT t.a, t.* exclude t.a FROM t`.
     // For simplicity, wildcard is involved.
-    QualifiedName(QualifiedName<'a>),
+    QualifiedName {
+        qualified: QualifiedName<'a>,
+        exclude: Option<ExcludeCol<'a>>,
```

Comment:
It seems too complicated, can we use `exclude: Option<Vec<Identifier<'a>>>`?

Reply:
I think it's ok.